src/erl/clojerl.erl

```diff
@@ -29,6 +30,27 @@ stop(_State) -> ok.
 %% Internal functions
 %%==============================================================================
 
+-spec init() -> ok.
+init() ->
+  CljeUserSym = clj_rt:symbol(<<"clje.user">>),
+  'clojure.core':'in-ns'(CljeUserSym),
+  %% This will not be available during bootstrap
+  case erlang:function_exported('clojure.core', refer, 2) of
+    true ->
+      ClojureCoreSym = clj_rt:symbol(<<"clojure.core">>),
+      'clojure.core':'refer'(ClojureCoreSym, []),
+
+      %% Maybe load user.clje script
+      clj_rt:load_script(<<"user.clje">>, false),
```

Comment:
The `user.clje` file doesn't need to include a leading `(ns clje.user)` form since the current ns is already that one.